ui/src/app/views/notebook-redirect/component.html

```diff
@@ -1,25 +1,43 @@
-<h2 *ngIf="notebookName">Loading Notebook '{{notebookName}}'</h2>
-<h2 *ngIf="!notebookName">Creating a New Notebook</h2>
-<div [ngSwitch]="progress" class="spinner-container">
-  <span class="spinner spinner-lg"></span>
-  <p *ngSwitchCase="Progress.Initializing" class="spinner-text">
-    <span class="spinner-header">Initializing notebook server</span><br>
-    May take up to 5 minutes
-  </p>
-  <p *ngSwitchCase="Progress.Resuming" class="spinner-text">
-    <span class="spinner-header">Resuming notebook server</span><br>
-    May take up to 1 minute
-  </p>
-  <p *ngSwitchCase="Progress.Authenticating" class="spinner-text">
-    Authenticating with the notebook server
-  </p>
-  <p *ngSwitchCase="Progress.Copying" class="spinner-text">
-    Copying notebook onto the notebook server
-  </p>
-  <p *ngSwitchCase="Progress.Creating" class="spinner-text">
-    Creating the new notebook
-  </p>
-  <p *ngSwitchCase="Progress.Redirecting" class="spinner-text">
-    Redirecting to the notebook server
-  </p>
-</div>
+<app-top-box>
+  <div *ngIf="progress !== Progress.Loaded">
+    <div style="display: flex; flex-direction: column; margin-left: 3rem; width: 780px; padding-top: 1rem">
```

Comment:
No action needed: We should probably make a codebase decision on this, but personally I'm not really a fan of inline styles (with some exceptions). They are less performant in the browser and it feels more consistent having styles all defined in the same place.